src/components/results/partial-results.vue

```diff
@@ -15,18 +15,18 @@
 </template>
 
 <script lang="ts">
-  import { PartialResultsList, PartialQueryButton } from '@empathyco/x-components/search';
   import { BaseGrid } from '@empathyco/x-components';
-  import { Component, Vue } from 'vue-property-decorator';
+  import { Component } from 'vue-property-decorator';
+  import HasQueryMixin from '../has-query.mixin.vue';
   import ResultComponent from './result.vue';
 
   @Component({
     components: {
       BaseGrid,
-      PartialResultsList,
-      PartialQueryButton,
-      Result: ResultComponent
+      Result: ResultComponent,
+      PartialResultsList: () => import('../search-has-query').then(m => m.PartialResultsList),
+      PartialQueryButton: () => import('../search-has-query').then(m => m.PartialQueryButton)
     }
   })
-  export default class PartialResults extends Vue {}
+  export default class PartialResults extends HasQueryMixin {}
```

Comment:
This component should also be lazily loaded. Partial results is a search feature, so you shouldn't need the mixin, or lazy loading components in this file.